proto/one/enterprise/data/enterprise_dataset_definitions.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
```

Comment:
We typically combine the singular and plural versions into a single file for our protobufs.  For an example, see https://github.com/AquaticInformatics/ONE.Interfaces.ProtocolBuffers/blob/develop/proto/one/common/library/parameter.proto#L54

Reply:
Joe mentioned this as well, and I just updated it to follow our pattern.